components/Layout/DatePicker.tsx

```diff
@@ -23,9 +26,39 @@ export default function HeaderDatePicker({
   maxDate,
 }: HeaderDatePickerProps) {
   const theme = useTheme()
+  const [locale, setLocale] = useState('en'); // default to English
+
+    useEffect(() => {
+      const handleLanguageChange = (event) => {
```

Comment:
Please provide type for _event_, because the build fails, without providing a type

Reply:
Fixed in https://github.com/TUM-Dev/TUMenu/pull/110/commits/42caffd2eef80ce7d4f0c6f315bd85b2f4feabee